libr/anal/p/anal_wasm.c

```diff
@@ -102,7 +102,7 @@ static int wasm_op(RAnal *anal, RAnalOp *op, ut64 addr, const ut8 *data, int len
 		case WASM_OP_LOOP:
 			//op->type = R_ANAL_OP_TYPE_CJMP;
 			addr2 = find_if_else (addr + op->size, data + op->size, len - op->size, true);
-			if (addr2 != UT64_MAX) {
+			if (addr2 != UT64_MAX && wasm_stack_ptr < WASM_STACK_SIZE) {
```

Comment:
this is quite error prone i think. if we end up using wasm_stack_ptr in other places, we are probably going to forget this change.

Reply:
why?

Reply:
hmm i agree

Reply:
just to be sure: does it make sense to stop analyzing the program if you reach the limit? or should it continue, skipping only the loop/block instructions? because if we should skip only loop/block instructions, maybe it still makes sense to have the check in those swith-cases (i don't know wasm :( )

Reply:
yes it does.

Reply:
I see... then just keep this PR :) maybe adding an error message.

Reply:
i'll make a new one later.